Extensions/read_posts.js

```diff
@@ -20,37 +20,37 @@ XKit.extensions.read_posts = new Object({
 
 	dashboard_regex: new RegExp("^https?://www.tumblr.com/dashboard(?:$|/)\\#?"),
 
-	last_undimmed: "",
+	undimmed_post: null,
+        currently_undimming: false,
```

Comment:
Oop. Meant to clean that before I pushed. *slaps own wrist* Will update; one sec.